docs/Researcher/cli-reference/runai-submit-dist-TF.md

```diff
@@ -273,6 +273,10 @@ runai submit-dist tf --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Configure a `Configmap` object for use in a container.
```

Comment:
> Configure a config 
sounds funny, doesn't it? you the expert
also not just a "use in a container" as there are other ways to use config map, suggest adding to the text something about it being a volume in the container

Reply:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

Reply:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```